deployment/ansible/ansible.cfg

```diff
@@ -0,0 +1,2 @@
+[defaults]
+inventory = inventory/
```

Comment:
The default inventory location is `/etc/ansible/hosts` on the local machine. Unfortunately, that's the only place Ansible looks without being overridden.